.github/workflows/test_environment.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         with:
             submodules: 'recursive'
       - name: test install environment ${{matrix.os}}.${{matrix.name}}
-        timeout-minutes: 45
+        timeout-minutes: 60
```

Comment:
why raising to 60 ? it should be a bit less 30min without cache . But not blocking